charts/redpanda/templates/configmap.yaml

```diff
@@ -71,7 +107,10 @@ data:
   {{- end }}
   {{- with (dig "cluster" dict .Values.config) }}
       {{- range $key, $element := .}}
-        {{- if $element }}
```

Comment:
We could do something like https://helm.sh/docs/chart_template_guide/function_list/#type-functions
how about `{{ if or (eq (typeOf $element) boolean) $element }}` that way we do not have to know an exhaustive list of boolean configs. We leave the responsibility of the user to know what they are doing and we offload maintenance.

Reply:
Sure I can change that.